### ISSUE TITLE
Alma Letters: a11y review tickets

### ISSUE THREAD

**Issue by rschwab** [UCSCLibrary/SiteImprove#118]
Email "letters" sent by Alma need to be checked and remediated for accessibility. The work of this ticket is to populate the board with tickets to accomplish the manual accessibility review for the letters. 

Sarah Troy, Moni, and Dawson should be given periodic updates, and can be consulted on for content questions.

[Spreadsheet of active letters and their priority](https://docs.google.com/spreadsheets/d/1lumL4EDrIMnE0COq5cdZ_rXW7zKLd8nr/edit?gid=847010505#gid=847010505)

- [x] Tickets created for the Alma letters manual accessibility review

**Comment by rschwab** [comment]
Background context: https://docs.google.com/document/d/1GffTNhHsWgv_VrwIWUL250-cOwxAeg_bivPFEMMCjMc/edit?disco=AAAB8E3WILk&usp_dm=true&tab=t.0#heading=h.wlg611ehvf0t
